src/main/kotlin/org/rust/ide/newProject/ui/AddUserTemplateDialog.kt

```diff
@@ -26,18 +27,18 @@ class AddUserTemplateDialog : DialogWrapper(null) {
     private val nameField: JBTextField = JBTextField()
 
     init {
-        title = "Add a Custom Template"
-        setOKButtonText("Add")
+        title = RsBundle.message("dialog.create.project.custom.add.template.title")
+        setOKButtonText(RsBundle.message("dialog.create.project.custom.add.template.action.add"))
         init()
     }
 
     override fun getPreferredFocusedComponent(): JComponent = repoUrlField
 
     override fun createCenterPanel(): JComponent = panel {
-        row("Template URL:") {
-            repoUrlField(comment = "A git repository URL to generate from")
+        row(RsBundle.message("dialog.create.project.custom.add.template.url")) {
+            repoUrlField(comment = RsBundle.message("dialog.create.project.custom.add.template.url.description"))
         }
-        row("Name:") {
+        row(RsBundle.message("dialog.create.project.custom.add.template.name")) {
```

Comment:
`:` disappeared after labels in the dialog panel. Is it intentional?
Personally, I'd prefer to keep them to be consistent with other dialogs in the platform